pkg/daemon/discover/discover.go

```diff
@@ -398,7 +398,11 @@ func updateDeviceCM(context *clusterd.Context) error {
 func probeDevices(context *clusterd.Context) ([]sys.LocalDisk, error) {
 	devices := make([]sys.LocalDisk, 0)
 	localDevices, err := clusterd.DiscoverDevices(context.Executor)
-	logger.Infof("localdevices: %+v", localDevices)
+	var devicesList []string
+	for _, device := range localDevices {
+		devicesList = append(devicesList, fmt.Sprintf("%+v", device))
+	}
+	logger.Infof("localdevices: %q", strings.Join(devicesList, ", "))
```

Comment:
You probably want this block after the error check.

Reply:
I would leave it before the error check as the `DiscoverDevices` can return with an error "anytime" a single device causes problems.
This would allow seeing what devices were discovered before that.

Reply:
Well if `clusterd.DiscoverDevices(context.Executor)` panics there is nothing to see, right?